src/containers/EntitySelectBoxContainer.js

```diff
@@ -8,27 +8,36 @@ const propTypes = {
   loadOptions    : PropTypes.func,
   onChange       : PropTypes.func,
   optionRenderer : PropTypes.func,
-  getOption      : PropTypes.func
+  getOption      : PropTypes.func,
+  getGnScores    : PropTypes.func
 }
 
 /**
  * This is the dumb presentation component for the Entity select box.
  */
 class EntitySelectBox extends Component {
   render () {
-    const { loadOptions, onChange, optionRenderer } = this.props
-    return <ReactSelect.Async cache={null}
-      filterOptions={false}
-      loadOptions={loadOptions}
-      onChange={onChange}
-      optionRenderer={optionRenderer} />
+    const { loadOptions, onChange, optionRenderer, getGnScores } = this.props
+    return <div className='row'>
+      <div className='col-md-8'>
+        <ReactSelect.Async cache={null}
+          filterOptions={false}
+          loadOptions={loadOptions}
+          onChange={onChange}
+          optionRenderer={optionRenderer}
+        />
+      </div>
+      <div className='col-md-4'>
+          <button className='btn btn-primary' onClick={() => getGnScores()}> Apply GeneNetworks Sorting</button>
```

Comment:
I'd think the sorting can happen live, is there a reason why we want the button press?